### PR TITLE
WAT-912 | Fix | Working hours are not being recorded

### PR DESCRIPTION
**Pull request by fernando-aloware**
## :memo:  Changes
**Changes :**
- Added logic to manage hours

<hr>

## :white_check_mark: Pull Request Review Checklist

- [ ] Variables have been given sensible, meaningful names. Try to check if full variable names are used instead of short forms.
- [ ] Code is [DRY (Don't Repeat Yourself)](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). (But make sure where this is applied, the two things are actually the same!)
- [ ] No values have been hard-coded where they should be drawn from constant files (frontend) or Models (backend).
- [ ] Comments are present where appropriate and provide useful information, not just stating what the code is obviously doing.
- [ ] Error handling is performed appropriately where things might fail.
- [ ] Logging is present, useful, and uses log levels appropriately.
- [ ] If component modifications are present, check all the instances that this component is used and make sure everything looks good and works as expected.
- [ ] If a generally used method is changed, check and make sure it doesn't break other apps and clients.
- [ ] Don't assume the code works - ask for a demo and test it yourself!
- [ ] The code follows the **Single Responsibility Principle**: Every module or class should have responsibility over a single part of the functionality provided by the software. More info: [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single-responsibility_principle).
- [ ] The code follows the **Open/Closed Principle**: Software entities should be open for extension, but closed for modification. This means the behavior of a module can be extended without modifying its source code. More info: [Open/Closed Principle](https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle).
- [ ] The code follows the **Liskov Substitution Principle**: Objects of a superclass should be replaceable with objects of a subclass without affecting the correctness of the program. More info: [Liskov Substitution Principle](https://en.wikipedia.org/wiki/Liskov_substitution_principle).
- [ ] The code follows the **Interface Segregation Principle**: Many client-specific interfaces are better than one general-purpose interface. This means no client should be forced to depend on methods it does not use. More info: [Interface Segregation Principle](https://en.wikipedia.org/wiki/Interface_segregation_principle).
- [ ] The code follows the **Dependency Inversion Principle**: High-level modules should not depend on low-level modules. Both should depend on abstractions. Moreover, abstractions should not depend on details. Details should depend on abstractions. More info: [Dependency Inversion Principle](https://en.wikipedia.org/wiki/Dependency_inversion_principle).

### CODE REVIEW

File: src/components/BusinessHoursDay.vue [hunk 1]
```diff
@@ -230,6 +241,11 @@ export default {
         return;
       }
 
+      //If change the opening hours, it is necessary to send the empty closing to add the default 9 hours
+      if(whichTime === 'open'){
```
Comment (jefferson-aloware) [inline src/components/BusinessHoursDay.vue]:
```suggestion
      if (whichTime === 'open') {
```

File: src/components/BusinessHoursDay.vue [hunk 2]
```diff
@@ -195,6 +197,14 @@ export default {
       el.style.visibility = binding.value ? 'visible' : 'hidden';
     }
   },
+  mounted: function() {
```
Comment (jefferson-aloware) [inline src/components/BusinessHoursDay.vue]:
```suggestion
  mounted: function () {
```

**Review by jefferson-aloware — CHANGES_REQUESTED** [review]

File: src/components/BusinessHoursDay.vue (hunk 2 above)
Comment (fernando-aloware) [reply src/components/BusinessHoursDay.vue]: @jefferson-aloware  thanks for the review, changes are applied 👍

File: src/components/BusinessHoursDay.vue (hunk 1 above)
Comment (fernando-aloware) [reply src/components/BusinessHoursDay.vue]: @jefferson-aloware  thanks for the review, changes are applied 👍

**Review by jefferson-aloware — APPROVED** [review]

**Review by keyner-aloware — APPROVED** [review]
Approved.

Just a small comment: '24' is repetitive. It would be better to create a new variable at the top of the component (in the script section) and use it across the component.

**Pull request merged**